wrappers/android/tools/camera/src/main/java/com/intel/realsense/camera/SettingsActivity.java

```diff
@@ -335,7 +399,7 @@ private void loadStreamList(Device device, List<StreamProfileSelector> streamPro
         ExpandableListView streamListView = findViewById(R.id.configuration_ex_list_view);
         HashMap<String, List<String>> expandableListDetail = new HashMap<String, List<String>>();
 
-        expandableListDetail.put("Configuration:(default-disable all)",settings_group);
+        expandableListDetail.put("Configuration:(turn off all to reset to default streams)",settings_group);
```

Comment:
What does it mean?
If the user want to disable all streams he will turn off all streams and than the default streams will be on?
How can he disable all streams in that case?
It's confusing.

Reply:
Oh OK, I didn't knew that was the case today.

Thanks for the informative explaination.